inference-engine/tests/ie_test_utils/functional_test_utils/layer_test_utils.hpp

```diff
@@ -115,6 +124,7 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     InferenceEngine::CNNNetwork cnnNetwork;
     InferenceEngine::InferRequest inferRequest;
     RefMode refMode = RefMode::INTERPRETER;
+    float precision = 1e-2;
```

Comment:
Can you please rename the attribute and corresponding setters and getters to `threshold` and `Set`\`GetThreshold` ? In most cases (at least in the tests) precision referees to computation precision but not to how precise comparison should be done